kustomize/components/clair/clair.deployment.yaml

```diff
@@ -45,8 +45,9 @@ spec:
             tcpSocket:
               port: 8080
           livelinessProbe:
-            tcpSocket:
-              port: 8080
+            httpGet:
+              port: clair-intro
```

Comment:
We don't currently set `introspection_addr` in Clair's config, will it still run the introspection server?

Reply:
Yes